bases/lefthook/biome/lefthook.yml

```
@@ -0,0 +1,14 @@
+# <files>の部分は対象としたいディレクトリを指定してください
+# gitでstageされたファイルのみを対象とする場合は{staged_files}を指定してください
+
+pre-commit:
+  commands:
+    fix:
+      glob: "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}"
+      run: npx biome check --apply --no-errors-on-unmatched --files-ignore-unknown=true <files> && git update-index --again
```

[imo]
設定に隠蔽できるなら可能な限りオプションは減らしたい気持ち

https://biomejs.dev/ja/reference/configuration/#filesignoreunknown

```suggestion
pre-commit:
  commands:
    fix:
      glob: "*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}"
      run: npx biome check --apply --files-ignore-unknown=true <files> && git update-index --again
```

---

これで秘匿できるのって`--files-ignore-unknown=true`ですよね
提示されたdiffだと`--no-errors-on-unmatched`が減ってるんですが、これはミスですか？

---

逆でした…
修正しました